app/common/uicontrols/programmanagement/views/patient-delete.html

```diff
@@ -0,0 +1,86 @@
+<div class="modal fade" id="deletePatientModal" tabindex="-1" role="dialog"
+     aria-labelledby="deletePatientModal" ng-controller="PatientCommonController as patientCommon">
+  <div class="modal-dialog modal-md" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+        <h4 class="modal-title" id="deletePatientModalLabel">{{'DELETE_PATIENT' | translate}}</h4>
+      </div>
+      <form class="form-horizontal" novalidate>
+        <div class="alert alert-danger alert-dismissible content-border-margin" role="alert"  ng-show="errorMessage">
+          <strong>{{errorMessage | translate}}</strong>
+        </div>
+
+        <div class="form-group form-group-lg">
```

Comment:
A simple checkbox field should be sufficient for the "Obitou" field. This toggle here is not consistent with the rest of the app.

![screenshot from 2017-09-18 12-13-08](https://user-images.githubusercontent.com/1554715/30538132-d52c8b8a-9c6c-11e7-8fa5-19038f945ef9.png)

Please use this one:

![screenshot from 2017-09-18 12-25-33](https://user-images.githubusercontent.com/1554715/30538166-fce3e826-9c6c-11e7-901b-0b937ebffd3a.png)